elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -20,8 +20,7 @@
 
 import { EventEmitter } from 'events';
 // tslint:disable-next-line no-require-imports
-import shuffle = require('lodash.shuffle');
-import { SCClientSocket } from 'socketcluster-client';
+import { shuffle } from 'lodash';
```

Comment:
maybe still keep `import shuffle = require('lodash.shuffle');` if not using any other function from lodash